test/jdk/jdk/dynalink/TypeConverterFactoryMemoryLeakTest.java

```diff
@@ -25,6 +25,11 @@
  * @test
  * @bug 8198540
  * @summary Test TypeConverterFactory is not leaking method handles
+ * @run main/othervm -XX:+UseG1GC TypeConverterFactoryMemoryLeakTest
+ * @run main/othervm -XX:+UseSerialGC TypeConverterFactoryMemoryLeakTest
+ * @run main/othervm -XX:+UseParallelGC TypeConverterFactoryMemoryLeakTest
+ * @run main/othervm -XX:+UseZGC TypeConverterFactoryMemoryLeakTest
+ * @run main/othervm -XX:+UseShenandoahGC TypeConverterFactoryMemoryLeakTest
```

Comment:
Ah, here is a test trivia. Some configurations do not have either ZGC or Shenandoah. So you need to check GC availabilty before adding `@run`. Usually done by splitting the `@test` blocks like this: https://github.com/openjdk/jdk/blob/master/test/hotspot/jtreg/serviceability/dcmd/gc/HeapDumpCompressedTest.java#L37-L107

Reply:
Thanks, that's an excellent suggestion! Implemented it now.